weather@mockturtl/files/weather@mockturtl/applet.js

```diff
@@ -1124,12 +1119,15 @@ MyApplet.prototype = {
     if (json.coord) {
       weather.coord.lat = json.coord.lat;
       weather.coord.lon = json.coord.lon;
+      weather.location.tzOffset = Math.round(json.coord.lon/15) * 3600;
```

Comment:
making sure we have the tz offset in the forecast, can't rely on the current weather call as both are async,